src/serviceFacades/filesystem.js

```diff
@@ -272,6 +272,25 @@ export const getPublicLinks = (key, paths) => {
     });
 };
 
+export const refreshCache = (key, { paths }) => {
+    return (
+        paths &&
+        paths.length > 0 &&
+        new Promise((resolve, reject) =>
+            Promise.all(
+                paths.map((path) =>
+                    callApi({
+                        endpoint: `/api/filesystem/refresh-cache?path=${path}`,
+                        method: "GET",
+                    })
+                )
+            )
+                .then((values) => resolve(values))
+                .catch((error) => reject(error))
+        )
+    );
```

Comment:
Pretty sure `Promise.all` doesn't need to be wrapped in another Promise.
```suggestion
            Promise.all(
                paths.map((path) =>
                    callApi({
                        endpoint: `/api/filesystem/refresh-cache?path=${path}`,
                        method: "GET",
                    })
                )
            );
```

Reply:
Ah, yep, found another example:
https://github.com/cyverse-de/sonora/blob/3071bf50a9a46b3dab7d383cc47a2150c1caba34/src/serviceFacades/tools.js#L153-L159

Reply:
Hmm. I was looking for example when I wrote this and found this way and then https://github.com/cyverse-de/sonora/blob/3071bf50a9a46b3dab7d383cc47a2150c1caba34/src/serviceFacades/analyses.js#L143 😕